docs/faq/install.txt

```diff
@@ -64,9 +64,9 @@ download page <https://www.python.org/downloads/>`_.
 
 Typically, we will support a Python version up to and including the first
 Django LTS release whose security support ends after security support for that
-version of Python ends. For example, Python 3.3 security support ended
-September 2017 and Django 1.8 LTS security support ended April 2018. Therefore
-Django 1.8 is the last version to support Python 3.3.
```

Comment:
I think it's worth updating. Can you change to the latest drop i.e. Python 3.8/3.9 and Django 5.0?

Reply:
Django 5.0 is not an LTS release.  We could do it with Django 4.2 which like 3.2 is an LTS release.

Reply:
> Django 5.0 is not an LTS release.

I know :) We drop support for Python 3.8 and 3.9 in Django 5.0, that's was I was talking about. So 4.2 is the last version that supports them and you should mention it.

Reply:
Is the failing test related to these changes?